yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -53,6 +54,8 @@
     </ol>
 
     <div class=outside_data>
+        <b>Tags: </b> {{ tags | join:", " }}
```

Comment:
I think it would be better to have each tag be an html element rather than a concatenated list of string

Reply:
Do you want them to remain on the same line? Or can they stack on top of each other

Reply:
i would say stay the same line, but wrap once it reaches the boundary. checkout [flexbox for css](https://www.w3schools.com/css/css3_flexbox.asp). we should make note this will be incompatible with some older browsers. I like to use [caniuse.com](https://caniuse.com/) to check browser compatibility with css elements

Reply:
@aniehuser done